lib/service/cache/redis.js

```diff
@@ -51,11 +51,22 @@ class Redis extends Service {
    * @returns {Promise}
    */
   _initSequence() {
+    // @deprecated @todo Remove those lines for Kuzzle v3 then 
+    // remove 'database' from .kuzzlerc.sample and default.config.js
+    if (this._config.database) {
+      this._config.options = { ...this._config.options, db: this._config.database };
+    }
```

Comment:
This makes the deprecated option take precedence over the newer one, this should be the other way around (see my previous comment)